__std__/flake.nix

```diff
@@ -0,0 +1,248 @@
+# The flake.nix is the entrypoint of all nix code.
+#
+# This repository uses the std tool https://github.com/divnix/std.
+# Familiarity with std is required to be able to contribute effectively.
+# While official documentation can be found in its GitHub, this flake
+# has been thoroughly commented so as to quickstart new maintainers.
+#
+# Most of what there is to know about the nix code inside this repository
+# can be learned by reading this file. A second read will be needed as some
+# std-specific terms may be referenced first and defined later.
```

Comment:
You can now link to the glossary in the docs.

See: https://divnix.github.io/std/glossary.html

Reply:
Added